### PR TITLE
Add deb packaging control files

### PR DESCRIPTION
**Pull request by oblitorum**

### CODE REVIEW

File: ssm-client.spec [hunk 1]
```diff
@@ -8,8 +8,8 @@ Version:        %{_version}
 Release:        1%{?dist}
 Group:          Applications/Databases
 License:        AGPLv3
-Vendor:         Percona LLC
-URL:            https://percona.com
+Vendor:         Shattered Silicon
+URL:            https://shatteredsilicon.net
```
Comment (oblitorum) [inline ssm-client.spec:12]: @gordan-bobic 
I'm not very sure about this, is it legal to change it to ours info? And similar one in deb control file -> https://github.com/shatteredsilicon/ssm-client/pull/14/files#diff-2a13f1344504379e877324d3a0375adcbcb4cb27d7e575ad8f4618a52d6a00e0R4

If it's not legal, I'll change it back

Comment (oblitorum) [reply ssm-client.spec:12]: Other packages (ssm-manage, ssm-server etc) don't have **Vendor** and use github repo url as the **URL**, not sure why they made a difference in this one.

Comment (oblitorum) [reply ssm-client.spec:12]: Sorry, forgot to quote the removed lines, full diff is here:

```diff
- Vendor:         Percona LLC
- URL:            https://percona.com
+ Vendor:         Shattered Silicon
+ URL:            https://shatteredsilicon.net
```

**Comment by gordan-bobic** [comment]
I think it's OK, pmm-client contains lots of code that didn't come from Percona, most of the code wasn't actually written by them. We could mention in the description that this is a fork of pmm-sclient from Percona, just to be extra safe and give credit where it is due.

File: ssm-client.spec [hunk 2]
```diff
@@ -8,15 +8,15 @@ Version:        %{_version}
...
 Source:         ssm-client-%{_version}.tar.gz
 AutoReq:        no
 BuildRequires:  glibc-devel, golang, unzip, gzip, make, perl-ExtUtils-MakeMaker, git
 
 %description
 Shattered Silicon Monitoring (SSM) is an open-source platform for managing and monitoring MySQL and MongoDB
-performance. It is developed by Percona in collaboration with experts in the field of managed database services,
+performance. It is a fork of Percona Monitoring and Management (PMM), which is developed by Percona in collaboration with experts in the field of managed database services,
```
Comment (oblitorum) [inline ssm-client.spec:19]: OK, does this sounds right?

**Comment by gordan-bobic** [comment]
How about:
```
+ performance. It is a fork of Percona Monitoring and Management (PMM), which is developed by Percona in collaboration with experts in the field of managed database services, and further improved by Shattered Silicon
```

File: ssm-client.spec [hunk 3]
```diff
@@ -8,16 +8,16 @@ Version:        %{_version}
...
 
 %description
 Shattered Silicon Monitoring (SSM) is an open-source platform for managing and monitoring MySQL and MongoDB
-performance. It is developed by Percona in collaboration with experts in the field of managed database services,
-support and consulting.
+performance. It is a fork of Percona Monitoring and Management (PMM), which is developed by Percona in collaboration
+with experts in the field of managed database services, and further improved by Shattered Silicon.
 SSM is a free and open-source solution that you can run in your own environment for maximum security and reliability.
 It provides thorough time-based analysis for MySQL and MongoDB servers to ensure that your data works as efficiently
 as possible.
```
Comment (oblitorum) [inline ssm-client.spec:23]: OK, changed it.

**Pull request merged**